packages/client/src/devices/CameraManager.ts

```diff
@@ -56,18 +56,14 @@ export class CameraManager extends InputMediaDeviceManager<CameraManagerState> {
   /**
    * Disables the video tracks of the camera
    */
-  pause() {
-    this.state.mediaStream?.getVideoTracks().forEach((track) => {
-      track.enabled = false;
-    });
+  pause(): Promise<void> {
+    return this.call.pauseTrack(TrackType.VIDEO);
```

Comment:
This won't work on the lobby screen

Reply:
yes I see.. so the way is to get the stream here and pause the tracks.. but send to sfu once joined then